CONTRIBUTING.md

```
@@ -0,0 +1,243 @@
+2. Run `/plugin marketplace add ./`
+3. Run `/plugin` then install the local plugin
+4. Test your tool
+5. Remove the local marketplace when done testing
+
+## Creating a New Tool
+
+### Skills
+
+Skills are the most full-featured tool type. Create a new directory under `helpers/skills/`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)